templates/_components/add-to-calendar.twig

```diff
@@ -0,0 +1,72 @@
+{%
+set classes = [
+  "localgov-add-to-calendar"
+]
+%}
+
+{{ attach_library('localgov_base/add-to-calendar') }}
+
+{% set lgdAddToCalendarAttributes = create_attribute() %}
+
+{% set ical_link %}data:text/calendar;charset=utf8,BEGIN:VCALENDAR%0AVERSION:2.0%0ABEGIN:VEVENT{% if atc_start_date %}%0ADTSTART:{{ atc_start_date|date("Ymd\\THi00\\Z") }}{% endif %}{% if atc_end_date %}%0ADTEND:{{ atc_end_date|date("Ymd\\THi00\\Z") }}{% endif %}{% if atc_title %}%0ASUMMARY:{{ atc_title }}{% endif %}{% if atc_details %}%0ADESCRIPTION: {{ atc_details|replace({'<br />': '\n', '<br>': '\\n', '<p>': '', '</p>': '\\n\\n'}) }}{% endif %}{% if atc_location %}%0ALOCATION: {{ atc_location|replace({'<br />': ' ', '<br>': ' ', '<p>': ' ', '</p>': ''})}}{% endif %}{% if atc_rrule %} %0ARRULE: {{ atc_rrule }} {% endif %}%0AEND:VEVENT%0AEND:VCALENDAR{% endset %}
```

Comment:
ical uses CRLF line endings so it should be `\r\n` instead of `\n` but then that doesn't mean the same thing as a line break. So text might will end up in a single line. Also and we certainly can ignore this, but line lengths should be 75 characters.

What is going to be more patchy for client support is having HTML in the description. Unless this has already been converted to plain text? Any text including HTML should be an additional version using ALTREP https://www.rfc-editor.org/rfc/rfc5545#section-3.2.1

Not important either but should probably output the date including the seconds, just in case someone (WHY?) it setting the seconds. Ymd\THis

Other comments would be enhancements:
 - We can add the geo location point `GEO:37.386013;-122.082932` which is lat;long
 - Doing it with timezone rather than Z would be nice, but lots of work.